src/workspace-hack/Cargo.toml

```diff
@@ -94,7 +94,7 @@ rand = { version = "0.8", features = ["small_rng"] }
 rand_chacha = { version = "0.3" }
 rand_core = { version = "0.6", default-features = false, features = ["std"] }
 regex = { version = "1" }
-regex-automata = { version = "0.4", default-features = false, features = ["dfa-onepass", "hybrid", "meta", "nfa-backtrack", "perf-inline", "perf-literal", "unicode"] }
+regex-automata = { version = "0.3", default-features = false, features = ["dfa-onepass", "hybrid", "meta", "nfa-backtrack", "perf-inline", "perf-literal", "unicode"] }
```

Comment:
Why it downgraded after cargo-hakari? 😇

Reply:
🤣🤣🤣 Hakari auto fixing will even revert the whole PR opened by dependent bot.

https://github.com/risingwavelabs/risingwave/pull/12575#issuecomment-1739401560

Reply:
Dependentbot is updating the dependency in `workspace-hack`, which is meaningless because this is generated file.